libsolidity/analysis/DeclarationTypeChecker.cpp

```diff
@@ -376,12 +376,17 @@ void DeclarationTypeChecker::endVisit(VariableDeclaration const& _variable)
 			if (_variable.isConstructorParameter())
 				errorString += " for constructor parameter";
 			else if (_variable.isCallableOrCatchParameter())
+			{
+				string visibility;
```

Comment:
Maybe
```suggestion
				string functionVisibility;
```
?

Reply:
Gotcha. I will update this and submit later tonight.